### ISSUE TITLE
HTML rendering of table of contents appears broken in Editor's copy

### ISSUE THREAD

**Issue by x448** [ace-wg/ace-dtls-profile#29]
The HTML rendering of TOC in working group's latest version (16) is OK.  

The [Editor's copy](https://ace-wg.github.io/ace-dtls-profile/) isn't rendering the TOC correctly in HTML.  "3.3. PreSharedKey Mode" is the first readable item in the TOC.

![image](https://user-images.githubusercontent.com/57072051/116133702-a6dffd80-a694-11eb-9edd-a2d05a720fa7.png)

**Comment by obgm** [comment]
Yes, at some point the tools (kramdown/xml2rfc) broke the layout. I suspect it has been in the updates for xml2rfc v3.

**Comment by obgm** [comment]
It looks as if the [i-d-template](/martinthomson/i-d-template) update in commit 79dfbad fixed this. Can you confirm?
